wpeverest/everest-forms
# Fix sortable scroll

### All Submissions:






### Changes proposed in this Pull Request:


Fixes scrolling issue when a field is drag and dropped to bottom of the builder area.

### How to test the changes in this Pull Request:

1. Create a long-form with fields in the builder area so that scroll is needed
2. Drag and Drop the top element to the bottom 
3. Check if automatic scroll triggers when field is dragged to the bottom of the builder.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Sortable Fields scrollable issue
